content/deploy-troubleshooting.md

```diff
@@ -18,13 +18,19 @@ Check these items if you're having trouble with uptime, performance or deploymen
   * If you run one container, that makes the machine your container is running on a single point of failure. In the event of a hardware failure, your app will be down until Galaxy starts it up again on a new machine.
 * Write in to <a href="mailto:support@meteor.com">support</a>. To minimize the back-and-forth, please send in the name of the affected app, the conditions that trigger the issue (confirmed or suspected), steps to reproduce, and relevant logs. Try to resolve errors listed in the logs before writing in.
   * If your app's container is running with the error, please try to leave it in the running state when contacting support, since that will help us to troubleshoot. Do not remove containers throwing errors, as that will prevent us from inspecting the container and the logs.
-  * While stopping the container is not as helpful for troubleshooting as leaving the container running, it will leave the logs intact.
+  * While stopping the container is not as helpful for troubleshooting as leaving the container running, it will leave the logs intact. Note that stopping your container will take down your app if it's the only container available.
 
 Note that code-level review lies outside the scope of Galaxy's support. If this is important to you, consider [Meteor Development Support](/support.html).
 
-<h2 id="503-errors">503 errors, 502 errors or Deployment Failures</h2>
+<h2 id="502-errors">502 errors</h2>
 
-Your app may throw a 503 error and show `Service Unavailable: No healthy endpoints to handle the request` when you try to visit your URL. You may also see a 502 error with the message `Registered endpoints failed to handle the request.` This means no healthy containers are currently available to serve your app.
+You may see a 502 error with the message `Registered endpoints failed to handle the request` when you try to visit your URL. This means that the request failed, despite the fact that our system thought there was a healthy container at the beginning of the request.
+
+This often happens because our back end wasn't able to respond, due to a deployment failure (described below).
```

Comment:
I've incorporated the s/our/your change into the document, in a new pull request, already merged.

https://github.com/meteor/galaxy-docs/pull/50